src/pykeen/losses.py

```diff
@@ -1081,7 +1081,7 @@ def prepare_negative_scores_for_softmax(
 
     # negative_scores have already been filtered in the sampler!
     # (dense) softmax requires unfiltered scores / masking
-    negative_scores_ = torch.zeros_like(batch_filter, dtype=negative_scores.dtype)
+    negative_scores_ = torch.zeros_like(batch_filter, dtype=negative_scores.dtype, device=negative_scores.get_device())
```

Comment:
what's the difference between `.device` and `.get_device()`?

Reply:
https://pytorch.org/docs/stable/generated/torch.Tensor.get_device.html seems to be doing something different